Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Media-Picker-3/index.md

```diff
@@ -0,0 +1,155 @@
+
+When this setting is enabled, a user who doesn't normally have access to the media selected as "Start Node" (/Design in this case), can access the media when using this particular Media Picker. If no Start node has been defined for this property any content can be viewed and selected of this property.
+
+
+### Image Croppings
+
+Define local image croppings, this data will stay locally, meaning that this will be stored on the document in this property.
+
+## Content Example
+
+![Media Picker Content](images/Media-Picker3-Content.jpg)
+
+## MVC View Example
+
+### Multiple enabled without Modelsbuilder
+
+```csharp
+@{
+    var typedMultiMediaPicker = Model.Value<IEnumerable<MediaWithCrops>>("medias");
+    foreach (var entry in typedMultiMediaPicker)
+    {
+        <img src="@entry.MediaItem.Url" style="width:200px"/>
+    }
+}
+```
+
+### Multiple enabled with Modelsbuilder
+
+```csharp
+@{
+    var typedMultiMediaPicker = Model.Medias;
+    foreach (var entry in typedMultiMediaPicker)
+    {
+        <img src="@entry.MediaItem.Url" style="width:200px" />
+    }
+}
+```
+
+## Multiple disabled without Modelsbuilder
+
+```csharp
+@{
+    var typedMediaPickerSingle = Model.Value<MediaWithCrops>("media");
+    if (typedMediaPickerSingle != null)
+    {
+        <img src="@typedMediaPickerSingle.MediaItem.Url" style="width:200px" alt="@typedMediaPickerSingle.MediaItem.Value("alt")" />
+    }
+}
+```
+
+## Multiple disabled with Modelsbuilder
+
+```csharp
+@{
+    var typedMediaPickerSingle = Model.Media;
+    if (typedMediaPickerSingle is MediaWithCrops mediaEntry)
+    {
+        <img src="@mediaEntry.MediaItem.Url" style="width:200px"/>
+    }
+}
+```
+
+## Using local crops
+
+Local image croppings are stored and retrived different than global crops, below is two examples of how to retrive local crops.
+
+### Using GetLocalCropUrl
+
+```csharp
+@{
+    foreach (var entry in Model.Medias)
+    {
+        <img src="@entry.GetLocalCropUrl("cropAlias")"/>
+    }
+}
+```
+
+### Using UrlHelper
+
+```csharp
+@{
+    foreach (var entry in Model.Medias)
+    {
+        <img src="@Url.GetCropUrl(entry.LocalCrops, "cropAlias")"/>
+    }
+}
+```
+
+## Use global crops
+
+Global image croppings are crops stored on the Media Item, by the Property Editor `Image Cropper`, making it shared between all usages of the media Item.
+
+The global crops are configured on the DataType of the `umbracoFile` property on the Media Type `Image`
+
+[Read about the Image Cropper here](../Image-Cropper/index.md)
+
+### Using GetCropUrl
+
+```csharp
+@{
+    foreach (var entry in Model.Medias)
+    {
+        <img src="@entry.MediaItem.GetCropUrl("cropAlias")"/>
+    }
+}
+```
+
+### Using UrlHelper
+
+```csharp
+@{
+    foreach (var entry in Model.Medias)
+    {
+        <img src="@Url.GetCropUrl(entry.MediaItem, "cropAlias")"/>
+    }
+}
+
+## Add values programmatically
+
+```This is not part of the current Media Picker v3, feel free to provide such.```
```

Comment:
What exactly does this part mean? 🤔
Does it mean that it is not possible to add values to this property programmatically?

Reply:
I just don't know how that's done, or whether more code needs to be made to enable such. But its part of MP2, so i thought to point that out.

Reply:
Good point! I've removed it for now. I will add it to the Docs issue tracker, in case someone in the community wants to help out with adding the section.